brownie/network/transaction.py

```diff
@@ -568,7 +572,11 @@ def _set_from_tx(self, tx: Dict) -> None:
             contract = state._find_contract(tx.get("to"))
             if contract is not None:
                 self.contract_name = contract._name
-                self.fn_name = contract.get_method(tx["input"].hex())
+                if HEXBYTES_LT_1_0_0:
```

Comment:
what if I just make a 2nd util in utils.hex that covers this case, and we just replace these instances with that helper to make everything cleaner?